File: backend/src/main/java/de/pawcheck/backend/cat/Cat.java
package de.pawcheck.backend.cat;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Cat {
    private String id;
    private String name;
    private Food wet;
    private Food dry;
    private String treats;
    private String catlery;
    private String water;
    private Toilet toilet;
    private Toy toy;

    //KONSTRUKTOR
    public Cat(String id, String name) {
        this.id = id;
        this.name = name;
    }
}

File: frontend/src/Pages/assets/Cat.ts
export type Cat = {
    id : string,
    name : string,
    dry? : Food,
    wet? : Food,
    treats? : string,
    catlery? : string,
    toilet? : Toilet,
    toy? : Toy,

}

export type Food = {
    morning? : string,
    evening? : string,
}

export type Toilet = {
    where? : string,
    howOften? : string,
    whereTheShit? : string,
}

export type Toy = {
    what? : string,
    where? : string,
}